Example/ViewController.m

```diff
@@ -218,25 +217,25 @@ - (void)getSessionFileList
     }];
 }
 
-- (void)ongoingAccessRetrieveData
+- (void)resumeOngoingAccess
 {
     [self resetClient];
     [self updateNavigationBarWithMessage:@"Retrieving Ongoing Access File List"];
-    [self.dmeClient authorizeOngoingAccessWithScope:nil oAuthToken:self.accessToken completion:^(DMESession * _Nullable session, DMEOAuthObject * _Nullable accessToken, NSError * _Nullable error) {
+    [self.dmeClient authorizeOngoingAccessWithScope:nil oAuthToken:self.oAuthToken completion:^(DMESession * _Nullable session, DMEOAuthToken * _Nullable accessToken, NSError * _Nullable error) {
```

Comment:
Please also update the variable name for `DMEOAuthToken` - from `accessToken` to `oAuthToken`